apiary.apib

```diff
@@ -2634,6 +2634,7 @@ Get a list of products.
         + data (object)
             + type: `product` (string)
             + id: `ee94b4c0-5786-0517-9d26-8dd1e6406f20` (string)
+            + purchase_price (Money, optional)
```

Comment:
B: This is on the `request` not in the `response` 😉

Reply:
No, the other way around. You added it in the response and it should be in the request.